receiver/prometheusreceiver/internal/otlp_metricfamily_test.go

```diff
@@ -227,8 +227,8 @@ func TestMetricGroupData_toSummaryUnitTest(t *testing.T) {
 				qn99 := qtL.AppendEmpty()
 				qn99.SetQuantile(.99)
 				qn99.SetValue(82)
-				point.SetTimestamp(14 * 1e6) // the time in milliseconds -> nanoseconds.
-				point.SetStartTimestamp(10 * 1e5)
+				point.SetTimestamp(14 * 1e6)      // the time in milliseconds -> nanoseconds.
+				point.SetStartTimestamp(14 * 1e6) // the time in milliseconds -> nanoseconds
```

Comment:
Use a constant for the 1e6 that better suggest the value.

Reply:
@bogdandrutu thanks. I have updated.